proxy-node-vsp-config/verify-service-provider.yml

```diff
@@ -28,11 +28,11 @@ hashingEntityId: ${HASHING_ENTITY_ID:-}
 verifyHubConfiguration:
   environment: ${VERIFY_ENVIRONMENT:-}
 
-samlSigningKey: ${SAML_SIGNING_KEY:-}
+samlSigningKey: ${VSP_SAML_SIGNING_KEY:-}
```

Comment:
Will the service work if this is blank? If not, why even have the fallback? Have dropwizard throw an error due to a missing interpolation token instead.